infra/modules/providers/azure/app-insights/README.md

```diff
@@ -47,9 +46,84 @@ module "app_insights" {
 }
 ```
 
+Note: Terraform will prompt for any variable values which are not passed into the `plan` or `apply` command, and for which default values are not set.
+
+Sample manual execution of the module using Terraform from within the `infra/modules/providers/azure/app-insights` directory:
+
+```
+c:\Users\user\repos\cobalt\infra\modules\providers\azure\app-insights>terraform apply
+var.appinsights_application_type
```

Comment:
You may want to include a reference to the `variables.tf` file here as a representation of the module's usage. This way we maintain a single source of truth with things like configuration descriptions and optional vs required.